src/components/activity/ActivityCountDown.tsx

```diff
@@ -19,7 +19,7 @@ const finishRemainingAnimations = (element: HTMLElement | null) => {
 const commitAnimationStyle = (element: HTMLElement | null) => {
   const animations = element?.getAnimations() ?? [];
   animations.forEach((a) => {
-    if (!element?.offsetParent === null) return;
```

Comment:
Good catch, I had seen errors on sentry before on this still, so I wonder if this change will fix it 👍